### DISCUSSION TITLE
5.0.0-beta.0

### DISCUSSION THREAD

**Discussion by awmleer** [ant-design/ant-design-mobile#3931] [Announcements]
本次发布没有包含新的改动，但是我们进入了 beta 阶段

<hr /><em>This discussion was created from the release <a href='https://github.com/ant-design/ant-design-mobile/releases/tag/v5.0.0-beta.0'>5.0.0-beta.0</a>.</em>

**Comment by agooddream** [comment]
期待早日支持SSR ，感谢大神们

**Comment by snakeUni** [comment]
期待早日支持SSR ，感谢大神们
